### PR TITLE
Add brew request-bottle

### PR DESCRIPTION
**Pull request by jonchang**
`brew request-bottle <formula>` to request a GitHub actions build.

### CODE REVIEW

File: cmd/brew-request-bottle.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#:  * `brew-request-bottle` <formula>
+#:    Submit a request to GitHub actions to build a bottle for Homebrew on Linux
+
+require "utils/github"
+
+module Homebrew
+  module_function
+
+  def formula
+    @formula ||= ARGV.last.to_s
```
[the remark this answers is not in the document]
Comment (issyl0) [reply cmd/brew-request-bottle.rb]: I just tried to do this. It needs a large rebase, and given this is your fork's `master` branch I don't feel comfortable force-pushing to that.

**Comment by dawidd6** [comment]
Can we have a separate step at the beginning of the whole dispatch job, to just print the formula name?
I see it is hard to differentiate which workflow run actually builds which formula. Need to look at `test-bot` output.

**Comment by dawidd6** [comment]
Also, maybe it would be helpful to somehow indicate who requested the bottle in commit message, that is pushed by LinuxbrewTestBot to master?

**Comment by issyl0** [comment]
@dawidd6 Yep. Absolutely this! I'll push more commits to #147 which supersedes this (for `Homebrew.args` usage)

**Pull request closed without merging**